integration_tests/mockApis/csipApi.ts

```diff
@@ -0,0 +1,178 @@
+import { stubFor } from './wiremock'
+
+const stubAreaOfWork = () => {
+  return stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: '/csip-api/reference-data/area-of-work',
+    },
+    response: {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json;charset=UTF-8',
+      },
+      jsonBody: [
+        {
+          code: 'A',
+          description: 'AreaA',
+          createdAt: new Date().toISOString(),
+          createdBy: 'foobar',
+        },
+        {
+          code: 'B',
+          description: 'AreaB',
+          createdAt: new Date().toISOString(),
+          createdBy: 'foobar',
+        },
+      ],
+    },
+  })
+}
```

Comment:
is it necessary to mock every single reference data type?
would it be okay if we just wildcard match /reference-data/<any-ref-data-type>, and return something generic like code: {'CODE1', descrption: 'Description One'}?

Reply:
It will make it much easier to check that the right values are coming through when we go to check your answers and can check that there is the specific description value there